### PR TITLE
Lockup transfer manager

### PR DESCRIPTION
**Pull request by victorwiebe**

### CODE REVIEW

**Comment by victorwiebe** [comment]
This blacklistTM branch is based on top of the lockupTM branch that is pending review.

File: src/contract_wrappers/modules/transfer_manager/lock_up_transfer_manager_wrapper.ts [hunk 1]
```diff
@@ -0,0 +1,804 @@
...
+
+// // Return types ////
+interface LockUp {
+  lockupAmount: BigNumber;
+  startTime: Date;
+  lockUpPeriodSeconds: BigNumber;
+  releaseFrequencySeconds: BigNumber;
+}
+
+interface GetLockUp extends LockUp {
```
Comment (monitz87) [inline src/contract_wrappers/modules/transfer_manager/lock_up_transfer_manager_wrapper.ts]: This should be called `LockUpWithAmount`. I know, horrible, but we should avoid naming interfaces based on which function returns them

File: src/contract_wrappers/modules/transfer_manager/lock_up_transfer_manager_wrapper.ts [hunk 2]
```diff
@@ -0,0 +1,804 @@
...
+  unlockedAmount: BigNumber;
+}
+
+interface GetLockupData extends GetLockUp {
```
Comment (monitz87) [inline src/contract_wrappers/modules/transfer_manager/lock_up_transfer_manager_wrapper.ts]: This should be called `LockUpData`

File: src/contract_wrappers/modules/transfer_manager/lock_up_transfer_manager_wrapper.ts [hunk 3]
```diff
@@ -0,0 +1,804 @@
...
+      });
+    }
+    return typedResult;
+  };
+
+  /**
+   * getListOfAddresses
+   */
+  public getListOfAddresses = async (params: LockupsParams): Promise<string[]> => {
+    assert.assert(params.lockupName.length > 0, 'LockUp Details must not be an empty string');
```
Comment (monitz87) [inline src/contract_wrappers/modules/transfer_manager/lock_up_transfer_manager_wrapper.ts]: Shouldn't this assertion say 'LockUp name must not be an empty string'?

File: src/contract_wrappers/modules/transfer_manager/lock_up_transfer_manager_wrapper.ts [hunk 4]
```diff
@@ -0,0 +1,804 @@
...
+    return {
+      lockupAmount: weiToValue(result[0], decimals),
+      startTime: bigNumberToDate(result[1]),
+      lockUpPeriodSeconds: result[2],
+      releaseFrequencySeconds: result[3],
+    };
+  };
+
+  /**
+   * getLockups
```
Comment (monitz87) [inline src/contract_wrappers/modules/transfer_manager/lock_up_transfer_manager_wrapper.ts]: You forgot to replace the placeholder comments (for most of the funcions)

**Pull request merged**